app/src/components/popups/BusStopPopup.vue

```diff
@@ -31,22 +31,29 @@
         </ul>
       </div>
 
-      <template v-if="stop.arrivals">
-        <template v-if="stop.arrivals.length > 0">
+      <template v-if="augmentedArrivals">
+        <template v-if="augmentedArrivals.length > 0">
           <router-link
-            v-for="arrival in stop.arrivals"
+            v-for="arrival in augmentedArrivals"
             :key="arrival.tripId"
-            class="flex py-2 w-full not-last:border-b-1 dark:border-dark-300"
+            class="flex flex-col py-2 w-full not-last:border-b-1 dark:border-dark-300"
             :to="{ name: 'map-marker', params: { markerType: 'bus', markerId: arrival.vehicleId } }"
           >
-            <i-fa-bus class="mr-2" />
-            <span class="mr-2">{{ arrival.routeName }}</span>
-            <span class="flex-grow">{{ arrival.direction }}</span>
-            <span>{{ eta(arrival) }}</span>
-            <div class="ml-2">
-              <i-fa-solid-clock v-if="arrival.state === 'planned'" />
-              <i-fa-solid-hand-paper v-if="arrival.state === 'stopping'" />
-              <i-fa-solid-running v-if="arrival.state === 'predicted'" />
+            <div class="flex flex-row">
+              <i-fa-bus class="mr-2" />
+              <span class="mr-2">{{ arrival.routeName }}</span>
+              <span class="flex-grow">{{ arrival.direction }}</span>
+              <span>{{ arrival.eta }}</span>
+              <div class="ml-2">
+                <i-fa-solid-clock v-if="arrival.state === 'planned'" />
+                <i-fa-solid-hand-paper v-if="arrival.state === 'stopping'" />
+                <i-fa-solid-running v-if="arrival.state === 'predicted'" />
+              </div>
+            </div>
+            <div class="flex flex-row gap-2 text-gray-500 dark:text-gray-400 text-xs">
+              <span>Next Stop:</span>
```

Comment:
```suggestion
              <span>Next stop:</span>
```

Should be coming from our translation file ;)